php/php.editor/src/org/netbeans/modules/php/editor/indent/CodeStyle.java

```diff
@@ -584,6 +584,10 @@ public boolean groupMulitlineArrayInit() {
         return preferences.getBoolean(GROUP_ALIGNMENT_ARRAY_INIT, getDefaultAsBoolean(GROUP_ALIGNMENT_ARRAY_INIT));
     }
 
+    public boolean groupMulitlineMatchArmArrow() {
```

Comment:
Typo, Mul[t]i[t]line

Reply:
https://github.com/apache/netbeans/blob/e8b206ef646754497a6890a1aa44ea2aa4cfcd8a/php/php.editor/src/org/netbeans/modules/php/editor/indent/CodeStyle.java#L579-L585

Thanks for catching it :) I've copied it from the above method. I'll also fix them.

Reply:
Fixed: https://github.com/apache/netbeans/compare/5271e2901f9a8c7421e518a0f5c9f8dc4ca7ea20..53dd6d320a66f0521e22e77fdb6f24a4c45eb30d